vtex-apps/bazaarvoice
# BV passive analytics events tracking

#### What is the purpose of this pull request?
Implements the Passive Analytic Events tracking described [here](https://developer.bazaarvoice.com/conversations-api/tutorials/bv-pixel/required-analytics), which is something required by BazaarVoice.

#### How should this be manually tested?

[Workspace](https://bazaarvoice--ecowaterqa.myvtex.com/whirlpool-central-water-filtration-system/p)
Open this workspace with the [Bazaarvoice chrome extension](https://chrome.google.com/webstore/detail/bazaarvoice-analytics-ins/olmofcadoappjkcnnhomdlnpmpbleekd) and check that there are events being tracked.

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/8443580/66508593-cf4ed980-eaa7-11e9-8ee7-f6e37015d7a6.png)

#### Types of changes
